CHANGELOG.md

```diff
@@ -13,6 +13,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+- To support fast join, update DefaultMeetingReadinessChecker's checkAudioConnectivity logic to check for audio packets sent and received metrics using metricsDidReceive observer in addition to attendee presence condition.
```

Comment:
What does fast join mean? Don't think we have any documentation mentions that term so you would want to explain in more details.